server/app/views/admin/questions/QuestionEditView.java

```diff
@@ -215,8 +215,21 @@ public Content renderViewQuestionForm(
 
   private Content renderWithPreview(
       Request request, DivTag formContent, QuestionType type, String title, Optional<Modal> modal) {
-    DivTag previewContent =
-        QuestionPreview.renderQuestionPreview(type, messages, applicantFileUploadRenderer);
+    DivTag previewContent;
+
+    if (settingsManifest.getNorthStarApplicantUi(request)) {
```

Comment:
Non-blocking question- This is part of Admin views. I am a little uncomfortable with using the same NorthStarApplicant Flag for Admin screens. If this feature is ready, we potentially want to globalize it before the ApplicantUI is fully ready. Since both applicant and admin are coupled, this will be hard to isolate on testing too.

Reply:
Good question. I think using `getNorthStarApplicantUi` is appropriate here since only the preview content will use North Star. The admin part of the page is still using J2HTML, and we don't intend to touch the admin side. That also means testing pre- and post- North Star is still possible.

(BTW, it's going to be months before North Star is ready to launch)

Reply:
What I mean is- if this feature - question preview using ThymeLeaf is ready, don't we want to go global with it irrespective of whether Applicant UI is ready or not?

Reply:
so these should be tied together, because we want the preview to look like what the applicant will see. It's not just that preview is in thymeleaf, it's that the preview shows the north star experience